modules/lobbies/scripts/list.ts

```diff
@@ -29,8 +32,16 @@ export async function run(
 	ctx: ScriptContext,
 	req: Request,
 ): Promise<Response> {
-	// TODO: Cache this without hitting the DO
+	await ctx.modules.captcha.guard({
+		key: "lobbies.list",
+		period: 20,
```

Comment:
period & requests should be configurable in config.ts and have defaults in the default config